app/microsoft/base.py

```diff
@@ -1,28 +1,15 @@
 import requests
 import logging
-from pprint import pformat
-
-from microsoft.helpers import create_client, get_token, BASE_URL, HTTP_HEADERS
-
-from django.conf import settings
 
+from microsoft.helpers import BASE_URL, HTTP_HEADERS
 
 logger = logging.getLogger(__name__)
 
 
 class BaseEndpoint:
     """Base class for MS Graph endpoints."""
 
```

Comment:
The part where we authenticate app with MS and get access token is now being done in __init__.py - this allows us to do it just once instead multiple times for multiple Endpoint objects.